test/appsignal/error_handler/error_matcher_test.exs

```diff
@@ -153,7 +153,7 @@ defmodule Appsignal.ErrorHandler.ErrorMatcherTest do
   test "Crashing GenServer with function error", %{fake_transaction: fake_transaction} do
     CrashingGenServer.start(:function_error)
 
-    :timer.sleep(20)
+    :timer.sleep(100)
```

Comment:
Is this an arbitrary value? Or actually linked to a GenServer timeout value?

Reply:
This is an arbitrary value that works on Travis. We should look into retrying this a couple of times to decrease the sleep time here, someday.